roles/ceph-defaults/tasks/ansible_logs.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: test if we have a local ansible.cfg file
+  local_action:
+    module: stat
+    path: ansible.cfg
+  register: ansible_cfg
+
+- name: lookup the value of log_path in ansible_logs.yml
+  local_action:
+    module: set_fact
+    ansible_log_location: "{{ lookup('ini', 'log_path section=defaults file=ansible.cfg') }}"
+  when: ansible_cfg.stat.exists == True
+
+- name: test if {{ ansible_log_location }} can be written
+  local_action: shell mktemp --dry-run --quiet --tmpdir=$(dirname {{ ansible_log_location }})
+  changed_when: false
+  failed_when: false
+  register: mktemp
+  when: ansible_cfg.stat.exists == True
+
+- name: fail if log_path directory from ansible.cfg is not writable
+  fail:
+    msg: "make sure log_path directory from ansible.cfg is writable by the user running ansible"
+  when:
+    - mktemp.stdout != '0'
+    - ansible_cfg.stat.exists == True
```

Comment:
shouldn't we stat for `{{ ansible_log_location }}` and check for `stat_ansible_log_location.stat.exists == False` instead ?